maintainers/maintainer-list.nix

```diff
@@ -15727,13 +15734,6 @@
     githubId = 219362;
     name = "Sarah Brofeldt";
   };
-  srid = {
```

Comment:
Why??

Reply:
oops, it slip because I use `git reset master --soft`
gonna fix it